srcs/utils/logging.cpp

```
@@ -42,3 +49,18 @@ void log(const std::string &message, LogLevel level)
 	std::string timestamp = get_timestamp();
 	std::cout << "[" << timestamp << "]:" << std::left << std::setw(20) << level_str << message << std::endl;
 }
+
+LogLevel parseLogLevel(const std::string &level)
```

cppcheck

[style] The function 'parseLogLevel' is never used.

(unusedFunction)